Makefile

```
@@ -124,6 +124,7 @@ verify: tidy download ## Verify code. Includes dependencies, linting, formatting
 	cp  $(KARPENTER_CORE_DIR)/pkg/apis/crds/* pkg/apis/crds
 	hack/validation/requirements.sh
 	hack/validation/labels.sh
+	MOD_DIRS=`echo "${MOD_DIRS}" | sed 's/.\/website//g'`
```

Would you be able to copy this to another environment variable so we make sure we're not editing the other calls here? 

Also, can you do the $(shell) syntax so that we follow the other in-line bash commands in the file? Just want to make sure we minimize bash weirdness.
```suggestion
	MOD_DIRS_VERIFY=$(shell echo $(MOD_DIRS) | sed 's/.\/website//g')
```